src/main/java/com/google/cloud/tools/maven/endpoints/framework/OpenApiDocsMojo.java

```diff
@@ -18,21 +18,17 @@
 package com.google.cloud.tools.maven.endpoints.framework;
 
 import com.google.api.server.spi.tools.EndpointsTool;
-import com.google.api.server.spi.tools.GetDiscoveryDocAction;
 import com.google.api.server.spi.tools.GetOpenApiDocAction;
 import com.google.common.base.Joiner;
 import com.google.common.base.Strings;
+import org.apache.maven.plugin.MojoExecutionException;
```

Comment:
looks like I never added the formatter/checkstyle plugins to this repo :(.

can you run `mvn com.coveo:fmt-maven-plugin:format` to make sure it's google java format compliant?

Reply:
The formatting modifies about 10 files. Would it be better done in another pull request so we don't lose the flavour of this commit and make review a bit easier? 

If not happy to add.

Reply:
Oh right, good point. Well I would like at least the new files to follow the format. Maybe just run it and then revert the files that we don't want included in the PR.

```
$ mvn com.coveo:fmt-maven-plugin:format
$ git checkout -- <files we want to exclude from this request>
```

or maybe just revert that "include" section.